usaspending_api/download/download_utils.py

```diff
@@ -6,42 +6,65 @@
 from usaspending_api.references.models import ToptierAgency
 
 
-def create_unique_filename(json_request, request_agency=None):
+def create_unique_filename(json_request, origination=None):
+    timestamp = datetime.strftime(datetime.now(timezone.utc), "%Y-%m-%d_H%HM%MS%S%f")
+    request_agency = json_request.get("agency", "all")
+
     if json_request.get("is_for_idv"):
-        download_name = "IDV_" + slugify_text_for_file_names(json_request.get("piid"), "UNKNOWN", 50)
+        slug_text = slugify_text_for_file_names(json_request.get("piid"), "UNKNOWN", 50)
+        download_name = f"IDV_{slug_text}_{timestamp}.zip"
     elif json_request.get("is_for_contract"):
-        download_name = "CONT_" + slugify_text_for_file_names(json_request.get("piid"), "UNKNOWN", 50)
+        slug_text = slugify_text_for_file_names(json_request.get("piid"), "UNKNOWN", 50)
+        download_name = f"CONT_{slug_text}_{timestamp}.zip"
     elif json_request.get("is_for_assistance"):
-        download_name = "ASST_" + slugify_text_for_file_names(json_request.get("assistance_id"), "UNKNOWN", 50)
-    else:
-        download_types = json_request["download_types"]
-        prefix = obtain_filename_prefix_from_agency_id(request_agency)
-        award_type_name = create_award_level_string(download_types)
-        download_name = "{}_{}".format(prefix, award_type_name)
-    timestamped_file_name = get_timestamped_filename("{}.zip".format(download_name))
-    return timestamped_file_name
+        slug_text = slugify_text_for_file_names(json_request.get("assistance_id"), "UNKNOWN", 50)
+        download_name = f"ASST_{slug_text}_{timestamp}.zip"
```

Comment:
This whole if forest could be avoided if you just had the slug text be conditional, and include "IDV" "CONT" or "ASST"

Reply:
I could clean this up a little, but how would I remove the if statement without a decent refactor?